daemon/emer-daemon.c

```diff
@@ -1446,6 +1515,7 @@ emer_daemon_finalize (GObject *object)
   g_warn_if_fail (priv->current_upload_cancellable == NULL);
 
   g_clear_pointer (&priv->aggregate_timers, g_hash_table_destroy);
+  g_clear_pointer (&priv->monitored_senders, g_hash_table_destroy);
```

Comment:
Is this the right order to clean these up?

After clearing `aggregate_timers`, `monitored_senders`'s values will contain GPtrArrays of `EmerAggregateTimerImpl`. But those arrays do not own a reference to those, so they are likely dangling pointers.

OTOH the cleanup function doesn't access them.